OWNERS

```diff
@@ -1,10 +1,10 @@
 # See the OWNERS docs at https://go.k8s.io/owners
-# See the OWNERS_ALIASES file at https://github.com/kubernetes-sigs/service-apis/blob/master/OWNERS_ALIASES for a list of members for each alias. 
+# See the OWNERS_ALIASES file at https://github.com/kubernetes-sigs/gateway-api/blob/master/OWNERS_ALIASES for a list of members for each alias.
 
 approvers:
   - sig-network-leads
-  - service-apis-admins
-  - service-apis-maintainers
+  - gateway-api-admins
+  - gateway-api-maintainers
```

Comment:
Should we do this at once? It could break our workflows if we don't get prow configurations fixed in the different repository.

Reply:
I see https://github.com/kubernetes/org/pull/2491. Great job!

Reply:
Thanks! This will be completed one kubernetes/org#2491 merges in. The owner aliases we rely on within this repo for approvals are all tied to the local OWNERS_ALIASES file which we simply try to keep in sync with k/org. So even if the timing doesn't match up perfectly, everything should still work.